sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1413,6 +1413,14 @@ object functions {
    */
   def bitwiseNOT(e: Column): Column = withExpr { BitwiseNot(e.expr) }
 
+  /**
+   * Get the value of the bit (0 or 1) at the specified position.
```

Comment:
let's copy the doc from the SQL expression.

Reply:
OK